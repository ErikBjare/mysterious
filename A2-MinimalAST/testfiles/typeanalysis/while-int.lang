int main() {
    while(1 == 2) {

    }

    while(1) {

    }
}
