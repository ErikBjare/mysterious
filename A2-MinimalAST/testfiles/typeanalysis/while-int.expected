Error at line 6: Expression is not compatible with boolean type.
